docs/Images.md

```diff
@@ -170,6 +170,44 @@ Beware that "flattening" images into JPEGs this way will fill transparent areas
 The allowed `image_filter` values are listed in the [image_parsing]( https://github.com/py-pdf/fpdf2/blob/master/fpdf/image_parsing.py) module and are currently:
 `FlateDecode` (lossless zlib/deflate compression), `DCTDecode` (lossy compression with JPEG) and `JPXDecode` (lossy compression with JPEG2000).
 
+## Output Intends ##
+
+> Output Intends [allow] the contents of referenced icc profiles to be embedded directly within the body of the PDF file. This makes the PDF file a self-contained unit that can be stored or transmitted as a single entity.
+
+### Add Desired Output Intent to the Output Intents Array ###
+`fpdf2` gives access to this feature through the method [`set_output_intent()`](https://py-pdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.set_output_intent):
+
+#### Specify ICCProfile Stream ####
+[`ICCProfileStreamDict`](https://py-pdf.github.io/fpdf2/fpdf/output.html#fpdf.output.output.ICCProfileStreamDict) Class is needed to specify the file object of the referenced icc profile.
+
+#### Example: ####
+```python
+from pathlib import Path
+from fpdf import FPDF
+from fpdf.enums import OutputIntentSubType
+from fpdf.output import PDFICCProfileObject
+
+HERE = Path(__file__).resolve().parent
+
+pdf = FPDF()
+
+with open(HERE / "sRGB2014.icc", "rb") as iccp_file:
+        icc_profile = PDFICCProfileObject(
+            contents=iccp_file.read(), n=3, alternate="DeviceRGB"
+        )
+
+pdf.set_output_intent(
+        OutputIntentSubType.PDFA,
+        "sRGB",
+        'IEC 61966-2-1:1999',
+        "http://www.color.org",
+        iccp_file,
+        "sRGB2014 (v2)",
+    )
```

Comment:
```suggestion
    icc_profile = PDFICCProfileObject(
        contents=iccp_file.read(), n=3, alternate="DeviceRGB"
    )

pdf.set_output_intent(
    OutputIntentSubType.PDFA,
    "sRGB",
    'IEC 61966-2-1:1999',
    "http://www.color.org",
    iccp_file,
    "sRGB2014 (v2)",
)
```
The indentation is slightly inconsistent.
I suggest to use 4 spaces, as it's the standard in this project.